dcap/types/src/quote3.rs

```diff
@@ -18,6 +18,9 @@ const QUOTE_SIZE: usize =
 // The offset to the authentication data
 const AUTH_DATA_OFFSET: usize = QUOTE_SIZE;
 
+// The offset to the report body for the app. From the start of the quote.
+const REPORT_BODY_OFFSET: usize = mem::size_of::<sgx_quote_header_t>();
```

Comment:
Is `sgx_quote_header_t` packed (or at least the same size on all the platforms we care to support)?

Reply:
Yes the `sgx_quote_header_t` is packed, it's inside of the `#pragma pack(push, 1)` in the header https://github.com/intel/SGXDataCenterAttestationPrimitives/blob/master/QuoteGeneration/quote_wrapper/common/inc/sgx_quote_3.h#L68